templates/interfaces.j2

```diff
@@ -1,49 +1,127 @@
 # vim: sw=4 ts=4 ft=jinja
 # {{ ansible_managed }}
 
+
+{% if network_management_pre_up is defined %}
+post_up {{ network_management_pre_up }}
+{% endif %}
+
 auto lo
 iface lo inet loopback
 
-{% for interface in bridges %}
+{% if network_management_clear_bridges %}
+# Delete all network bridges
+pre-up for br in `ovs-vsctl list-br`; do ovs-vsctl --if-exists "$br"; done
+{% endif %}
+
+################################################################################
+# Configure all network interfaces                                             #
+################################################################################
+{% for interface in interfaces %}
+{% if interface.bring_up|default(network_management_default_bring_up) %}
+auto {{ interface.name }}
+{% endif -%}
+
```

Comment:
These blocks belong together, please delete this empty line

Reply:
Required by jinja to to add at least one newline!!!